src/qudi/gui/time_series/time_series_gui.py

```diff
@@ -299,6 +304,20 @@ def on_deactivate(self):
 
         self._mw.close()
 
+    @_view_settings.representer
+    def __repr_view_settings(self, value):
```

Comment:
I've seen this a few times now in some other code, yet I don't really know why the core requires the representer to have "value" here. In most cases it is not used. Can you enlighten me about this @timoML or @Neverhorst?

Reply:
`StatusVar` will pass the currently stored object in the instance variable to the representer. If your status variable at runtime is for example a custom object for which the type can not be stored in a YAML file, you can use the representer to convert this object to a simple type. This could look like e.g.:
```python
class MyCustomObject:
    def __init__(a, b, c):
        self.a = a
        self.b = b
        self.c = c


class MyModule(LogicBase):
    
    _my_status_var = StatusVar(name='my_status_var', default=list())
        
    @_my_status_var.representer
    def _repr_my_status_var(self, value):
        return [value.a, value.b, value.c]
        
    @_my_status_var.constructor
    def _constr_my_status_var(self, value):
        return MyCustomObject(*value)
```

In fact getting the value of the `StatusVar` from somewhere in the module (i.e. from `self`) is actually bad design since the value of the `StatusVar` at runtime does not correspond to the one being dumped by the representer in that case. But it's possible...